scripts/ssl-manager.js

```diff
@@ -135,6 +141,49 @@ function SSLManager(config) {
         return resp;
     };
 
+    me.checkUpdateExpiration = function() {
+        var LE_TEXT = "Let's Encrypt auto update",
+            HTML = "html/update-expired.html",
+            UPDATE_DECREASED = "updateDecreased",
+            UPDATE_DISABLED = "updateDisabled",
+            timeStamp;
+
+        timeStamp = Date.parse(nodeManager.jemSslCheckdomain().replace(/-/g, "/"));
+
+        if (!config[UPDATE_DISABLED] && me.isExparedToDays(timeStamp, REMOVE_UPDATE_DAYS)) {
+            me.disableAutoUpdate();
+            me.updateSettingsValue(UPDATE_DISABLED, true);
+
+            return me.sendEmail("Disable " + LE_TEXT, HTML, {
+                SUPPORT_EMAIL : SUPPORT_EMAIL,
+                DAYS: String(REMOVE_UPDATE_DAYS),
+                ACTION: "Auto-update retries were disabled."
+            });
+        }
+
+        if (!config[UPDATE_DECREASED] && me.isExparedToDays(timeStamp, DECREASE_UPDATE_DAYS)) {
+            me.exec([
+                [ me.scheduleAutoUpdate, "0 0 " + Math.floor(Math.random() * (16 -8) + 8) + " * *" ],
+                [ me.updateSettingsValue, UPDATE_DECREASED, true ]
+            ]);
+
+            return me.sendEmail("Decrease " + LE_TEXT, HTML, {
+                SUPPORT_EMAIL : SUPPORT_EMAIL,
+                DAYS: String(DECREASE_UPDATE_DAYS),
+                ACTION: "The frequency of auto-update retries was decreased to once per month."
+            });
+        }
+
+        return { result: 0 }
+    };
+
+    me.isExparedToDays = function(timestamp, days) {
+        var currentDate = Date.now(),
```

Comment:
use new Date.getTime() for backward compatibility

Reply:
done